runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/AnagramClue.java

```diff
@@ -71,85 +71,85 @@
 		new AnagramClue("ICY FE", "Fycie", new WorldPoint(2630, 2997, 0), "East Feldip Hills"),
 		new AnagramClue("I DOOM ICON INN", "Dominic Onion", new WorldPoint(2609, 3116, 0), "Nightmare Zone", "How many reward points does a herb box cost?", "9,500"),
-		new AnagramClue("I EVEN", "Nieve", new WorldPoint(2432, 3422, 0), "The slayer master in Gnome Stronghold", "How many farming patches are there in Gnome stronghold?", "2"),
-		new AnagramClue("IM N ZEZIM", "Immenizz", new WorldPoint(2592, 4324, 0), "The Imp inside Puro-Puro"),
-		new AnagramClue("KAY SIR", "Sir Kay", new WorldPoint(2760, 3496, 0), "The courtyard in Camelot Castle", "How many fountains are there within the grounds of Camelot castle?", "6"),
-		new AnagramClue("LEAKEY", "Kaylee", new WorldPoint(2957, 3370, 0), "Rising Sun Inn in Falador", "How many chairs are there in the Rising Sun?", "18"),
-		new AnagramClue("LARK IN DOG", "King Roald", new WorldPoint(3220, 3476, 0), "Ground floor of Varrock castle", "How many bookcases are there in the palace library?", "24"),
-		new AnagramClue("LOW LAG", "Gallow", new WorldPoint(1805, 3566, 0), "Vinery in the Great Kourend", "How many vine patches can you find in this vinery?", "12"),
+		new AnagramClue("I EVEN", "Nieve", new WorldPoint(2432, 3422, 0), "Gnome Stronghold, near Stronghold Slayer Cave entrance", "How many farming patches are there in Gnome stronghold?", "2"),
+		new AnagramClue("IM N ZEZIM", "Immenizz", new WorldPoint(2592, 4324, 0), "Puro-Puro"),
+		new AnagramClue("KAY SIR", "Sir Kay", new WorldPoint(2760, 3496, 0), "Camelot Castle courtyard", "How many fountains are there within the grounds of Camelot castle?", "6"),
+		new AnagramClue("LEAKEY", "Kaylee", new WorldPoint(2957, 3370, 0), "Falador pub", "How many chairs are there in the Rising Sun?", "18"),
+		new AnagramClue("LARK IN DOG", "King Roald", new WorldPoint(3220, 3476, 0), "Ground floor, Varrock castle", "How many bookcases are there in the palace library?", "24"),
+		new AnagramClue("LOW LAG", "Gallow", new WorldPoint(1805, 3566, 0), "Hosidius vinery", "How many vine patches can you find in this vinery?", "12"),
 		new AnagramClue("LADDER MEMO GUV", "Guard Vemmeldo", new WorldPoint(2447, 3418, 1), "Gnome Stronghold Bank", "How many magic trees can you find inside the Gnome Stronghold?", "3"),
-		new AnagramClue("MAL IN TAU", "Luminata", new WorldPoint(3508, 3237, 0), "Near Burgh de Rott entrance"),
-		new AnagramClue("MACHETE CLAM", "Cam the Camel", new WorldPoint(3300, 3231, 0), "Outside Duel Arena", "How many items can carry water in Gielinor?", "6"),
-		new AnagramClue("ME IF", "Femi", new WorldPoint(2461, 3382, 0), "Gates of Tree Gnome Stronghold"),
+		new AnagramClue("MAL IN TAU", "Luminata", new WorldPoint(3508, 3237, 0), "Burgh de Rott, near general store"),
+		new AnagramClue("MACHETE CLAM", "Cam the Camel", new WorldPoint(3300, 3231, 0), "Al Kharid, south-west of the Duel Arena", "How many items can carry water in Gielinor?", "6"),
```

Comment:
This seems wordier and less accurate than the previous "Outside Duel Arena" description. Generally Cam is only just west of the duel arena, rarely south-west. If any more specification is needed for this area, maybe it should be "Outside Duel Arena entrance"